egs/wsj/s5/steps/libs/common.py

```diff
@@ -321,10 +321,19 @@ def get_feat_dim_from_scp(feat_scp):
     return feat_dim
 
 
-def split_data(data, num_jobs):
-    run_kaldi_command("utils/split_data.sh {data} {num_jobs}".format(
-                        data=data,
-                        num_jobs=num_jobs))
+def split_data(data, num_jobs, per_utt=False, per_reco=False):
```

Comment:
You never need these options now, and the per-reco stuff wouldn't work; please revert this change.